Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG LLVM_VERSION=800
+ARG LLVM_VERSION=900
```

Comment:
I'm ok with 9 being default, but 8 support should hopefully work once Anvill is built for 8